src/Response.php

```diff
@@ -168,23 +167,23 @@ public function loadXml($response, int $libXmlOptions = 0)
             return false;
         }
 
-        if (! empty($xml->fault)) {
+        if (isset($xml->fault) && is_object($xml->fault)) {
             // fault response
             $this->fault = new Fault();
             $this->fault->setEncoding($this->getEncoding());
             $this->fault->loadXml($response);
             return false;
         }
 
-        if (empty($xml->params)) {
+        if (! isset($xml->params)) {
             // Invalid response
             $this->fault = new Fault(652);
             $this->fault->setEncoding($this->getEncoding());
             return false;
         }
 
         try {
-            if (! isset($xml->params) || ! isset($xml->params->param) || ! isset($xml->params->param->value)) {
+            if (! isset($xml->params->param) || ! isset($xml->params->param->value)) {
```

Comment:
Isn't it possible to do this in one `isset`? Suchas `! isset($xml->params->param, $xml->params->param->value)`